help_to_heat/frontdoor/views.py

```diff
@@ -539,8 +539,8 @@ def handle_post(self, request, session_id, page_name, data, is_change_page):
         request_supplier = request_data.get("supplier")
         if request_supplier == "Bulb, now part of Octopus Energy":
             next_page_name = "bulb-warning-page"
-        if request_supplier in unavailable_suppliers:
-            next_page_name = "applications-closed"
+        if request_supplier == "British Gas":
```

Comment:
Is there a reason we totally remove the old `request_supplier in unavailable_suppliers` logic to have something unique to british gas?

Reply:
I think it's going to be redundant logic at the moment since British Gas and Utility Warehouse are the only unavailable suppliers and so we're not making use of the applications-closed page, although if you'd rather we kept it in as a catch-all should we add more unavailable suppliers in the future, I can add it back in!

Reply:
No, I agree with you, if we don't use it at any point it should be removed! (YAGNI)

Reply:
Great, sounds good, thanks!

Reply:
I'd actually go the opposite way on this: proliferating extra views for every supplier that's unavailable isn't great. The plan was actually already to move Utility Warehouse to use the same view as the "standard" one we were using for British Gas, with that view being slightly more dynamic and some logic to vary what is shown as required.

I'd still rather we take that approach here: one template that changes the content based on the supplier, rather than distinct views for each.